src/infra/docs/docs-rs/README.md

```diff
@@ -0,0 +1,14 @@
+# docs.rs
+
+* Source code: [rust-lang/docs.rs][repo]
+* Hosted on: `docsrs.infra.rust-lang.org` (behind the bastion -- [how to connect][bastion-connect])
+* Maintainers: [Joshua Nelson], [Pietro Albini]
+* [Instance metrics][grafana-instance] (only available to infra team members).
+* [Application metrics][grafana-app] (only available to infra team members).
+
+[repo]: https://github.com/rust-lang/docs.rs
+[grafana-instance]: https://grafana.rust-lang.org/d/rpXrFfKWz/instance-metrics?orgId=1&var-instance=docsrs.infra.rust-lang.org:9100
+[grafana-app]: https://grafana.rust-lang.org/d/-wWFg2cZz/docs-rs?orgId=1
+[bastion-connect]: https://github.com/rust-lang/infra-team/blob/master/docs/hosts/bastion.md#logging-into-servers-through-the-bastion
+[Joshua Nelson]: https://github.com/jyn514
```

Comment:
Ideally we'd just link to the website but I couldn't create a permalink to the docs.rs team section.
```suggestion
[docs.rs team]: https://github.com/rust-lang/team/blob/master/teams/docs-rs.toml
```

Reply:
I think this permalink should work: https://www.rust-lang.org/governance/teams/dev-tools#docs-rs